src/commands/admin/refresh.ts

```diff
@@ -0,0 +1,60 @@
+import { BOT } from '@root/config';
+import { BOTMASTER_PERMS } from '@root/src/lib/permissions';
+import { Command } from '@root/src/lib/types/Command';
+import { readdirRecursive } from '@root/src/lib/utils';
+import { ApplicationCommandData, CommandInteraction } from 'discord.js';
+
+export default class extends Command {
+
+	description = `Re-loads all of ${BOT.NAME}'s commands. WARNING: This takes forever`;
```

Comment:
Description should also warn of the rate limiting because I feel like that's really important to note

Reply:
This is a good idea but I feel this can be addressed after we merge this, since the need to get this feature in is somewhat high. I do agree though since it will make it clear to new admins that it is a long process.

Reply:
Why not just do it now? It's only a description change. I can do it if you want?

Reply:
I'm pretty sure everyone is around right now to re-approve anyways, to be fair